public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,123 @@
+# metrics diff
+
+Show a table of changes in [metrics](/doc/command-reference/metrics#description)
+between two versions of the <abbr>DVC repository</abbr>.
+
+> This command requires that the <abbr>project</abbr> is a
+> [Git](https://git-scm.com/) repository.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v]
+                        [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref     Git reference to the older version to compare
```

Comment:
rev now?

Reply:
Yes, per iterative/dvc/pull/3299. OK, I'll apply those doc changes in this PR as well...

Reply:
Done!